cmd/clusterctl/examples/openstack/ubuntu/machines.yaml.template

```diff
@@ -42,5 +43,6 @@ items:
         floatingIP: 129.114.111.153
         securityGroups:
         - default
+        userDataSecret: worker-user-data
```

Comment:
Might change if you decide to change type of userDataSecret to `SecretReference`. See other comment.